### PR TITLE
chore: change existing featureflags to use config

### LINKED ISSUE #901: Convert existing flags to use config.ts
- Convert existing FeatureFlags to use config.ts
- Remove any flags that aren't in use anymore

### PR DESCRIPTION
**Pull request by mk1r**
closes #901 

also removes the need of `useFeature` function

### CODE REVIEW

File: src/constants/FeatureFlagEnum.ts [hunk 1]
```diff
@@ -1,15 +1,8 @@
 import { getConfig } from 'config'
 
-export enum FlagValue {
-  On = 'on',
-  Off = 'off'
-}
-
 export type Flag = typeof FeatureFlagEnum[keyof typeof FeatureFlagEnum]
 
 export const FeatureFlagEnum = {
```
Comment (0xdef1cafe) [inline src/constants/FeatureFlagEnum.ts:5]: while you're at it, do you wanna rename this to just `FeatureFlag`?

**Review by 0xdef1cafe — APPROVED** [review]
approved with suggested change

File: src/constants/FeatureFlagEnum.ts (hunk 1 above)
Comment (gomesalexandre) [reply src/constants/FeatureFlagEnum.ts:5]: Was going to comment the exact same :D digged in the codebase and except for a few cases, we seem to mostly use the naming without `Enum` prefix.

**Review by gomesalexandre — APPROVED** [review]
🍬

**Review by theoboldfrazier — APPROVED** [review]

File: src/constants/FeatureFlagEnum.ts (hunk 1 above)
Comment (mk1r) [reply src/constants/FeatureFlagEnum.ts:5]: yeah sure, both the flag and enum?

Comment (0xdef1cafe) [reply src/constants/FeatureFlagEnum.ts:5]: yes please

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**